### PR TITLE
Create basic flask app structure

### PR DESCRIPTION
**Pull request by jpmartinspt**
## Done
Create initial structure to start porting the application to flask.

## QA
- ./run
- go to http://0.0.0.0:8017 and see a `401`
- go to http://0.0.0.0:8017/?token=correcthorsebatterystaple and see a `200 {}`

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -36,13 +36,10 @@ jobs:
...
           name: Install pip dependencies
-          command: pip3 install -r requirements.txt && pip install gunicorn
-      - run:
-          name: Install mongo
-          command: apt-get install -y mongodb
+          command: pip3 install -r requirements.txt
       - run:
           name: Run the site server
-          command: gunicorn webapp.wsgi -b 0.0.0.0:80
+          command: talisker.gunicorn webapp.app:app -b 0.0.0.0:80
```
Comment (jkfran) [inline .circleci/config.yml]: I know it's not important but in other projects we use instead `./entrypoint 0.0.0.0:80`

Comment (jpmartinspt) [reply .circleci/config.yml]: good spot. i'll update

File: .circleci/config.yml [hunk 2]
```diff
@@ -53,4 +50,4 @@ workflows:
     jobs:
       - lint-python
       - test-python
-      - test-site
+#     - test-site
```
Comment (nottrobin) [inline .circleci/config.yml]: Why commented out? 'Cos it's currently broken?

Comment (jpmartinspt) [reply .circleci/config.yml]: Because `/` is returning a 401, I will try to use `/_status/check`.

Comment (nottrobin) [reply .circleci/config.yml]: Testing that `/` returns a 401 would be another option, either seems fine.

**Pull request merged**